AnkiDroid/src/main/java/com/ichi2/libanki/DB.java

```diff
@@ -60,7 +60,7 @@
     private boolean mMod = false;
 
     /**
-     * Open a database connection to an ".anki" SQLite file.
+     * Open a database connection to an ".anki2" SQLite file.
```

Comment:
Why not future proof the thing and just say 
```suggestion
     * Open a connection to the SQLite collection database.
```

But yes - boring commits are quick, while interesting ones are sadly slower :-)